cairo-oracle-hint-processor/src/rpc_1_hint_processor/mod.rs

```diff
@@ -96,11 +98,35 @@ impl<'a> Rpc1HintProcessor<'a> {
             .post(self.server.as_ref().unwrap())
             .json(&data)
             .send()
+            .map_err(|_| {
+                format!(
+                    "Error sending request to oracle server {}",
+                    self.server.as_ref().unwrap()

```

Comment:
nit: we can assing a server variable here: https://github.com/reilabs/cairo-hints/pull/11/files#diff-d4c1b22be77210c463bb5cc500303c4a7c129752d5ac1e4ec9a5ad425d7d3648R85 and avoid further unwraps here and in line 112.